shared/gh/js/views/tenant-admin/gh.admin-manage-orgunits.js

```diff
@@ -0,0 +1,260 @@
+/*!
+ * Copyright 2015 Digital Services, University of Cambridge Licensed
+ * under the Educational Community License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['gh.core', 'gh.constants', 'jquery.jeditable'], function(gh, constants) {
+
+    /**
+     * Add an organisational unit
+     *
+     * @private
+     */
+    var addOrgUnit = function() {
+        // Get the form data
+        var displayName = $('#gh-manage-orgunits-modal-text').val();
+        var type = $('#gh-manage-orgunits-modal-type').val();
+        var parentId = parseInt($('#gh-manage-orgunits-add-modal').attr('data-id'), 10) || null;
+        var groupId  = parseInt($('#gh-manage-orgunits-add-modal').attr('data-groupid'), 10) || null;
+
+        // Create the organisational unit
+        gh.api.orgunitAPI.createOrgUnit(gh.data.me.AppId, displayName, type, parentId, groupId, null, null, false, function(err, data) {
```

Comment:
I don't think this is right. Each course, subject and part should have a different group. IIRC we need to be able to assign different users to different parts. Take for example:
```
Asian and Middle eastern studies
  + Chinese
      + Part IA   (Dr Miranda Foo)
      + Part IB   (Dr Miranda Foo)
      + Part II   (Dr Jack Bower)
      + Part III   (Dr Jack Bower)
   Japanese
      + Part IA   (Dr Miranda Foo)
      + Part IB   (Dr Miranda Foo)
      + Part II   (Dr Jack Bower)
      + Part III   (Dr Jack Bower)
```

If the part always inherits the subject/course group we won't be able to make these distinctions